grafana-dashboard/Kafka_dashboard.json

```diff
@@ -0,0 +1,3228 @@
+{
+  "__inputs": [
+    {
+      "name": "DS_CLICKHOUSE-OPERATOR-PROMETHEUS",
```

Comment:
could you replace 
`DS_CLICKHOUSE-OPERATOR-PROMETHEUS`
to 
`DS_PROMETHEUS`
in whole dashboard?

Which allow us to use this dashboard inside deploy/prometheus?